src/db.rs

```diff
@@ -54,11 +56,12 @@ pub struct DB {
 }
 
 impl DB {
-    pub fn new() -> Self {
-        let connection = sqlite::open("DB.db").unwrap();
-        connection
-            .execute(
-                "CREATE TABLE IF NOT EXISTS messages(
+    pub fn new() -> Lazy<Self> {
```

Comment:
But is't `DB::new()` only called once in `main`?

Reply:
Indeed 🤦‍♂️  Gonna fix

Reply:
Removed the last commit. It is indeed pointless